main.py

```diff
@@ -17,8 +17,7 @@
 
 if __name__ == "__main__":
 
-    update_database_task(url_price, url_volume, 1800)
-    database_thread = threading.Thread(target=update_database_task)
+    database_thread = threading.Thread(target=update_database_task, kwargs={'url_price': url_price, 'url_volume': url_volume, 'update_delay': 1800})
```

Comment:
I'd also rename your update_database_task name to be more descriptive about what it is doing